### PR TITLE
[Appendix] Add crawl opt-out (closes #498)

### LINKED ISSUE #498: Describe how developers can block web crawlers
There may be a case where a web app developer does not want their web app to be crawled.  I was thinking proposing an additional element to support this, but realized the web already has a way to block crawlers.  

Below is some draft language, if this seems reasonable, please 👍 and I'll submit a PR.

> Like other web resources, a web app manifest should be accessible to any web browser or web crawler.
>  
> If a web app developer wants to block traffic from web crawlers, the developer MAY do so by including the web app manifest in a robots.txt file.  This is further described in the [robots.txt](http://www.robotstxt.org/) protocol.

### PR DESCRIPTION
**Pull request by RobDolinMS**
Adding an Appendix section of developer guidance including how developers can opt-out of web crawl.

Also adds a placeholder for some additional examples.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3597,5 +3597,32 @@ <h3 id="applying">
...
+        Developer Guidance
+      </h2>
+      <p>
+        This section provides non-normative guidance for developers of web app manifest files 
+      </p>
+      <h3>
+        Additional Examples
+      </h3>
+      <p>
+        TODO: Add additional examples
```
Comment (kenchris) [inline index.html]: I think there is a special way to add the TODOs with respec

**Comment by marcoscaceres** [comment]
I'd still prefer we do this in an explainer.md file. This stuff gets out of date pretty quick, so it's better to have it in file that we can update independently of the spec.

**Comment by RobDolinMS** [comment]
Thanks @marcoscaceres.  I'll re-propose to add as a related explainer.md file instead of as an appendix.  

Thanks also to @MichaelChampion for explaining some of the normative vs. non-normative issues.

File: index.html [hunk 2]
```diff
@@ -3597,5 +3597,32 @@ <h3 id="applying">
...
+      </p>
+      <h3>
+        Interaction With Web Crawlers
+      </h3>
+      <p>
+        Like other web resources, a web app manifest should be accessible to any web browser or web crawler.
+      </p>
+      <p>
+        If a web app developer wants to block traffic from web crawlers,
```
Comment (alrra) [inline index.html]: I personally think that  _`"wants to block traffic"`_ is not the correct wording here, and instead, something in the lines of _`"wants to inform"`_ should be used.

From http://www.robotstxt.org/faq/nosecurity.html: 

> .../robots.txt is not intended for access control, so don't try to use it as such. Think of it as a "No Entry" sign, not a locked door.

**Comment by RobDolinMS** [comment]
Replaced by PR #505

File: index.html (hunk 2 above)
Comment (RobDolinMS) [inline index.html]: Thanks @alrra.  I've replaced this PR with #499 and incorporated your suggestion of "block traffic" to "inform"

**Pull request closed without merging**